charts/umbrella/Chart.yaml

```
@@ -28,7 +28,7 @@ sources:
   - https://github.com/eclipse-tractusx/tractus-x-umbrella
 
 type: application
-version: 2.1.0
+version: 2.1.1
```

I'd raise a minor for more intuitive versioning
```suggestion
version: 2.2.0
```